lib/class-wp-theme-json-schema-v1-to-v2.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Class that implements a WP_Theme_JSON_Schema migrator.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that implements a migration from a v1 data structure into a v2 one.
+ */
+class WP_Theme_JSON_Schema_V1_To_V2 implements WP_Theme_JSON_Schema {
```

Comment:
Classes should have _Gutenberg suffix otherwise when we pass the class with the same name to core we are going to have a crash.